### DISCUSSION TITLE
`FluentSwitch` `::slotted()` Css selector doesn't work unless `!important` is applied

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [microsoft/fluentui-blazor#2032] [Q&A] [answered]
I am attempting to use the `FluentSwitch` control, and all works well except for the styling:

![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/53d596da-460c-4dfa-a3c8-c06ce6d3d3a1)

I found the selector:

![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/5d9f1cbc-834f-48fa-9067-fb4726dbdc53)

But only when I add `!important` does it work:
![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/1b827166-652c-4fe4-bc29-c17b15e53cc1)

![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/7d28b21d-2420-4bb9-95c2-eb8107d43294)

Is this a setting somewhere?  Or a bug?  I wanted to verify before opening another issue.  For reference I "fixed" this by encountering this blog:
https://www.matuzo.at/blog/2022/100daysof-day45

EDIT: I should also mention that `margin-inline-start` does not return for any other style applied to this control:
![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/91dc8c95-ed71-4435-9b2a-7a3eb7989601)

So it is unclear to me how this is not being applied and what is overriding it.  If it was overridden it seems like this rule would be crossed out altogether, but it is not.

**Comment by vnbaaij** [comment, marked as answer]
If I look at the default example at https://www.fluentui-blazor.net/Switch, it looks like the styling is being applied to the 'new feature' switch. I'm not at a pc for a couple of days so can't look further into it. Can you compare your code with the example?
Oh, BTW the 'constructed' reference as to where the css is defined, indicated that it comes from the web component script. If it is indeed a bug, we might need to open an issue in that repo.

**Reply by Mike-E-angelo** [reply]
Thank you for your time and answer @vnbaaij it got me looking around a bit further.  I was looking for the `margin-inline-start` but there is a rule applied to the template that I am using that is applying to all margins:

![image](https://github.com/microsoft/fluentui-blazor/assets/2931376/ee9e4fc6-bcd4-4679-b7ba-ec00df964009)

It seems like this would strike out the `::slotted` rule to help assist in tracking down, but it does not.  Seems like a firefox issue.   In any case, removing this rule from the theme fixed it. 👍
